bundles/CoreBundle/Resources/config/authenticator_security.yaml

```diff
@@ -3,8 +3,8 @@ pimcore:
     security:
         factory_type: password_hasher
         password_hasher_factories:
-            Pimcore\Bundle\AdminBundle\Security\User\User: pimcore_admin.security.password_hasher_factory
+            Pimcore\Security\User\User: pimcore_admin.security.password_hasher_factory
 
 scheb_two_factor:
     security_tokens:
-        - Pimcore\Bundle\AdminBundle\Security\Authentication\Token\TwoFactorRequiredToken
\ No newline at end of file
+        - Pimcore\Bundle\AdminBundle\Security\Authentication\Token\TwoFactorRequiredToken
```

Comment:
So we need the TwoFactorRequiredToken only in the admin bundle right?

Reply:
yes, this should be moved to AdminBundle

Reply:
done. also moved the password hasher to core https://github.com/pimcore/pimcore/pull/14918/commits/4ac858302ccf1a589600d39662f89b372898caff